storage/mempool.go

```diff
@@ -60,6 +61,11 @@ func pendingStateKey(stateID uint32) []byte {
 	)
 }
 
+func decodePendingStateKey(keyBytes []byte) uint32 {
+	lastFour := keyBytes[len(keyBytes)-4:]
+	return binary.BigEndian.Uint32(lastFour)
+}
```

Comment:
Do we have these badger key formats documented somewhere? Would be nice to have an overview of how the badger keyspace is structured.

Reply:
The closest we have is this: https://github.com/worldcoin/hubble-commander/blob/18ed7591cdcaca3db918395a8e5489381a5ffaca/docs/badger/data_structures.md#L1

You're right it'd be nice to have an overview somewhere, the best way would be to keep all the keys as consts and functions in a single file which gets imported, I'll play with that when I have some downtime